modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItCommonApiTest.java

```diff
@@ -111,37 +103,27 @@ public void testSessionExpiration() throws Exception {
     @Test
     public void checkTimestampOperations() {
         String kvTblName = "tbl_all_columns_sql";
-        String schemaName = "PUBLIC";
-        String keyCol = "key";
-        int maxTimePrecision = TemporalColumnType.MAX_TIME_PRECISION;
+        String keyCol = "KEY";
 
         Ignite node = CLUSTER_NODES.get(0);
 
         // TODO: https://issues.apache.org/jira/browse/IGNITE-19162 Trim all less than millisecond information from timestamp
         //String tsStr = "2023-03-29T08:22:33.005007Z";
-        String tsStr = "2023-03-29T08:22:33.005Z";
+        // TODO: IGNITE-20105 it should be "2023-03-29T08:22:33.005Z";
```

Comment:
I just stole this code from catalog-feature branch where guys already had to solve this issue, see https://github.com/apache/ignite-3/blob/catalog-feature/modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItCommonApiTest.java

I'm not sure it's a good idea to fix it here: this will just increase the amount of conflicts.